about.html

```diff
@@ -0,0 +1,292 @@
+								<a class="nav-link text-light" href="#">English</a>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link text-light" href="#">My Page</a>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link text-light" href="#">Log out</a>
+							</li>
+						</ul>
+					</div>
+
+					<nav class="navbar navbar-expand-sm">
+						<div class="container-fluid bg-white">
+							<a class="navbar-brand text-dark" href="index.html"
+								><img
+									class="victoria-logo"
+									src="./asset/images/victoria-university-logo.png"
+									alt="Victoria-university-logo" />
+							</a>
+
+							<ul
+								class="navbar-nav conatiner-fluid d-flex justify-content-between align-items-center">
+								
+								<li class="nav-item mx-4" id="desktop-nav">
+									<a class="nav-link text-dark" href="./index.html">Home</a>
+								</li>
+								<li class="nav-item mx-4" id="desktop-nav">
+									<a
+										class="nav-link text-dark"
+										id="desktop-nav"
+										href="./about.html"
+										>About</a
+									>
+								</li>
+								<li class="nav-item mx-4" id="desktop-nav">
+									<a class="nav-link text-dark" href="#">Program</a>
+								</li>
+								<li class="nav-item mx-4" id="desktop-nav">
+									<a class="nav-link text-dark" href="#">Sponsor</a>
+								</li>
+								<li class="nav-item mx-4" id="desktop-nav">
+									<a class="nav-link text-dark" href="#">Log out</a>
+								</li>
+								<li class="nav-item mx-4 border border-5 border-danger">
+									<a class="nav-link text-center text-danger" href="#"
+										>Our legacy</a
+									>
+								</li>
+							</ul>
+						</div>
+					</nav>
+				</div>
+			</header>
+
+			<main>
+				<!--Intro section -->
+				<section >
+					<div class="hero-container py-3">
+						<h1 class="h1 text-danger text-center mb-3 hero-about-content">
+							VICTORIA UNIVERSITY ALUMNI STUDENT SUMMIT 2023
+						</h1>
+						<div class="bg-white p-2 hero-intro-container">
+							<p class="hero-intro text-center border border-5 border-white ">
+								We are proud that so many of our alumni choose to give something
+								back to the university, not only through financial donations but
+								by providing practical support and career guidance to current
+								students which equally gives the latter a head start as one enters
+								the world of work.
+							</p>
+						</div>
+
+						<div class=" p-4 text-center text-dark shadow mb-2" >
+							<p>
+								Please contact us per Email for nay further questions about
+								VICTORIA UNIVERSITY ALUMNI STUDENT SUMMIT 2023!
+							</p>
+							<p>admissions@vu.ac.ug /marketing@vu.ac.ug</p>
+						</div>
+					</div>
+
+					<div class=" py-2 ">
+						<h1 class="h1 text-danger text-center mb-3 shadow p-4 mb-2 ">
+							VU ALUMNI STUDENT SUMMIT 2023 CREATIVE LOGO 
+						</h1>
+						<div class="d-flex justify-content-center"><hr class="bar bg-danger"> </hr></div>
```

Comment:
[REQUIRED] Kindly ensure that this text matches. the template in terms of colour and sizing. This is a requirement